source/features/toggle-files-button.tsx

```diff
@@ -33,17 +33,26 @@ function addButton(): void {
 async function toggleHandler(): Promise<void> {
 	const isHidden = select('.repository-content')!.classList.toggle('rgh-files-hidden');
 	await (isHidden ? cache.set(cacheKey, true) : cache.delete(cacheKey));
+
+	// Remove notice after the first click
+	select('.rgh-files-hidden-notice')?.remove();
 }
 
 async function init(): Promise<Deinit[]> {
 	const repoContent = (await elementReady('.repository-content'))!;
+
 	if (await cache.get<boolean>(cacheKey)) {
 		repoContent.classList.add('rgh-files-hidden');
+		select('#files')!.after(
+			<div className="py-1 text-right text-small rgh-files-hidden-notice" style={{paddingRight: '19px'}}>
+				Files hidden <ArrowUpIcon className="v-align-middle"/>
```

Comment:
Maybe write, `unhide files`?

Reply:
This is meant to be a notice of the current state of the UI, not a call to action. Maybe it can be extended to "The file list was collapsed via Refined GitHub" and make it in gray text.

Reply:
+1 for a longer and gray text